### ISSUE TITLE
Missing theme configuration file

### ISSUE THREAD

**Issue by facelessuser** [squidfunk/mkdocs-material#519]
## Description

Mkdocs material does not have a `mkdocs_theme.yml`. I run my mkdocs build with strict in Travis to catch whether my mkdocs environment has warnings etc. It is currently failing because of this.  This is due to the mkdocs `0.17` release.

```
WARNING - The theme 'material' does not appear to have a configuration file. Please upgrade to a current version of the theme.
```

An example theme config: https://github.com/mkdocs/mkdocs/blob/master/mkdocs/themes/mkdocs/mkdocs_theme.yml

Not sure if any options need to be specified in it.  Probably more that it is missing, but maybe we need to specify something in it. Not sure yet.

It's not a critical issue, but it should probably get added for correctness.

### Expected behavior

Material should not generate warnings.

### Actual behavior

Is generating the warning that the config file is not found.

### Steps to reproduce the bug

1. Build with strict:  `mkdocs build --clean --verbose --strict`

### Package versions

* Python: Any
* MkDocs: `0.17`
* Material: Any

### Project configuration

NA

### System information

* OS: ANY
* Browser: NA

**Comment by squidfunk** [comment]
Thanks for reporting. 0.17 also broke the search. Will provide a configuration file asap.

**Comment by squidfunk** [comment]
**Should** be fixed in master. Can you verify please? I've some problems with getting 0.17 to work locally, probably messed up my Python installation.

**Comment by squidfunk** [comment]
There are some fundamental changes with 0.17. Hope we'll get everything to work, but I will probably do 2.0.0 with breaking changes.

**Comment by facelessuser** [comment]
Sounds good! I expect some breaking changes as we approach mkdocs 1.0.

**Comment by squidfunk** [comment]
Released with 1.12.1
